Docker/itksoftwareguide-bin/README.md

```diff
@@ -1 +1,21 @@
+Description
+-----------
+
 Holds the volume with the build for the Software Guide.
+
+Volumes
+-------
+
+Source folder: /home/itk/src/ITKSoftwareGuide
+Binary folder: /home/itk/bin/ITKSoftwareGuide
+
+Usage
+-----
+
+Build in the docker container:
+  ./run.sh itksoftwareguide-bin
+
+Use ITKSoftwareGuide outside of the docker container:
+  ./run.sh itksoftwareguide-bin \
+    -v ${HostITKSoftwareGuidSourceDirectory}:/home/itk/src/ITKSoftwareGuide \
+    -v ${HostITKSoftwareGuidBinaryDirectory}:/home/itk/bin/ITKSoftwareGuide
```

Comment:
@jcfr could you clarify by what you mean by 'this'? if you mean the documentation on what version of CMake support SSL, that seems odd as now as all versions on the download page do support it. 
It might be worthwhile looking at updating https://cmake.org/cmake/help/v3.9/command/file.html documentation to spell out when SSL came to each OS ( it varies ).

As far as explicitly listing the GLIBC version, that seems valuable. I think adding it to the paragraph above the downloads would be good.